lib/media/ver20/set_audio_encoder_configuration.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Onvif.Media.Ver20.SetAudioEncoderConfiguration do
+  import SweetXml
+  import XmlBuilder
+
+  alias Onvif.Device
+  alias Onvif.Media.Ver10.Profile.AudioEncoderConfiguration
```

Comment:
There isn't any change in the audio encoder configuration from ver10 apart from them supporting all the encoding from https://www.iana.org/assignments/media-types/media-types.xhtml#audio. So just reused it.